src/main/resources/templates/application/summary/application-linked-case-search-results.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>LAA CCMS - Search for an application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+    <div id="breadcrumbs">
+        <a href="/application/summary/linked-cases/search" class="govuk-back-link">Return to Linked Case Search</a>
+    </div>
+    <div id="main-content">
+        <span class="govuk-caption-l">Search for an application or case</span>
```

Comment:
Is it not a search for cases only?

Reply:
Nope you can search unsubmitted applications too, just not itself